### ISSUE TITLE
New Package Update

### ISSUE THREAD

**Issue by deepanshushuklad11** [DylanVann/react-native-fast-image#1040]
DreamSportsLab has forked this repo and created new package .
 [@d11/react-native-fast-image](https://www.npmjs.com/package/@d11/react-native-fast-image)
Dream11 is actively maintaining this new package and will be fixing relevant and reproducible issues of this repository.

**Comment by Yann-prak** [comment]
Do you need help maintaining this?

**Comment by deepanshushuklad11** [comment]
> this?

Much appreciated

**Comment by wogno** [comment]
Hi, this package applied this commit [1032](https://github.com/DylanVann/react-native-fast-image/pull/1032)

**Comment by deepanshushuklad11** [comment]
Not yet .

**Comment by deepanshushuklad11** [comment]
> Hi, this package applied this commit [1032](https://github.com/DylanVann/react-native-fast-image/pull/1032)

Not yet

**Comment by geo-vi** [comment]
do you have a new github repository or fork that I can star?

**Comment by deepanshushuklad11** [comment]
> do you have a new github repository or fork that I can star?

Yes https://github.com/dream-sports-labs/react-native-fast-image
